src/chat/chatter.py

```diff
@@ -273,43 +274,52 @@ def updatePlayer(self):
-            str(int(player.ladder_rating_deviation))))
+            str(int(player.ladder_rating_deviation)))
 
         league = player.league
         if league is not None:
-            formatting = ("Division : {}\n"
-                          "Global Rating: {}")
-            self.rankItem.setToolTip(formatting.format(
-                league["division"],
-                str(int(rating))))
-            self.rankItem.setIcon(util.THEME.icon("chat/rank/%s.png" % league["league"]))
+            icon_str, tooltip_str = league["league"], "Division : " + league["division"] + "\n" + tooltip_str
         else:
-            self.rankItem.setIcon(util.THEME.icon("chat/rank/newplayer.png"))
-
-        self._verifySortOrder()
+            icon_str = "newplayer"
+        self.rankItem.setIcon(util.THEME.icon("chat/rank/%s.png" % icon_str))
+        self.rankItem.setToolTip(tooltip_str)
 
     def updateGame(self):
         # Status icon handling
         game = self.user_game
-        player = self.user_player
         if game is not None and not game.closed():
-            url = game.url(player.id)
+            formatting = "<br/>title: {}<br/>mod: {}<br/>map: {}<br/>players: {} / {}<br/>id: {}"
+            game_str = formatting.format(game.title, game.featured_mod, maps.getDisplayName(game.mapname),
+                                         str(game.num_players), str(game.max_players), str(game.uid))
             if game.state == GameState.OPEN:
-                self.statusItem.setIcon(util.THEME.icon("chat/status/lobby.png"))
-                self.statusItem.setToolTip("In Game Lobby<br/>"+url.toString())
+                if game.password_protected:
+                    game_str = " (private) Game Lobby:</b>" + game_str
+                else:
+                    game_str = " Game Lobby:</b>" + game_str
+                if game.host == self.user.name:
+                    icon_str, tooltip_str = "host", "<b>Hosting"
+                else:
+                    icon_str, tooltip_str = "lobby", "<b>In"
             elif game.state == GameState.PLAYING:
-                self.statusItem.setIcon(util.THEME.icon("chat/status/playing.png"))
-                self.statusItem.setToolTip("Playing Game<br/>"+url.toString())
-        else:
-            self.statusItem.setIcon(QtGui.QIcon())
-            self.statusItem.setToolTip("Idle")
+                if game.has_live_replay:
+                    icon_str, tooltip_str = "playing", "<b>Playing:</b>"
+                else:
+                    icon_str, tooltip_str = "playing5", "<b>Playing:</b>" + "   LIVE DELAY (5 Min)"
+            else:  # game.state == something else
+                icon_str, tooltip_str = "unknown", "<b>Playing maybe ...</b>"
+
+        else:  # game is None or closed
+            icon_str, tooltip_str, game_str = "none", "Idle", ""
+
+        self.statusItem.setIcon(util.THEME.icon("chat/status/%s.png" % icon_str))
+        self.statusItem.setToolTip(tooltip_str + game_str)
```

Comment:
This method became a bit too convoluted IMO, it's hard to follow the logic with multiple indentations and icon + tooltip logic mixed together. We could do the icon logic separately from the tooltip logic. I also don't like how "Playing" or "Game Lobby" are duplicated - let's give it some descriptive name like "tooltip_header"and add info like 'live delay' or 'private game' separately, maybe even on a separate line.

Actually, maybe we can operate on lists of lines at first, then just join the list with `</br>`.

Reply:
I rearanged it with header und game separate and in order